phpstan-baseline.neon

```diff
@@ -75,6 +75,11 @@ parameters:
 			count: 1
 			path: src/bundle/Controller/BulkOperation/BulkOperationController.php
 
+		-
+			message: "#^Access to protected property Ibexa\\\\Contracts\\\\Core\\\\Repository\\\\Values\\\\User\\\\LookupLimitationResult\\:\\:\\$hasAccess\\.$#"
+			count: 4
+			path: src/bundle/Controller/Content/ContentTreeController.php
+
```

Comment:
This is a bug in core. It can be mitigated by using `hasAccess()` method call for the relevant part. Requires https://github.com/ibexa/core/pull/349.